### PR TITLE
Restore Wings upgrading docs.

### PR DESCRIPTION
**Pull request by Crell**
Per Customer Care request.

It may need updating as Wings has evolved, but that is up to CC.

### CODE REVIEW

File: src/dedicated/overview/upgrading.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+* Branch from the Staging and Production environments
+* Synchronize data from the Staging and Production environments
+* Snapshot the Staging and Production environments
+
+### Branching hierarchy
+
+Before converting your project, the branches include a repository for Development, Staging, and Production. Each repository has a master branch with deployment targets configured for Staging and Production.
+
+After converting your project, the hierarchical relationships appear in your Project Web Interface with two, main environment branches for Staging, and Production:
```
Comment (chadwcarlson) [inline src/dedicated/overview/upgrading.md]:
```suggestion
After converting your project, the hierarchical relationships appear in your Project Web Interface with two, main environment branches for Staging and Production:
```

File: src/dedicated/overview/upgrading.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+![Production and Staging are special children of master](/images/wings-branches.png)
```
Comment (chadwcarlson) [inline src/dedicated/overview/upgrading.md]: Broken image here, which I don't think got committed. Should probably go in `/images/dedicated` as well

Comment (chadwcarlson) [inline src/dedicated/overview/upgrading.md]:
```suggestion
![Production and Staging are special children of master](/images/dedicated/wings-branches.png)
```

File: src/dedicated/overview/upgrading.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+If you have additional code, such as new extensions in your Production environment without following this workflow, then deployments from Integration or Staging overwrite your Production code.
+
+### Verify user account access
+
+We recommend verifying your user account access and permissions set in the Integration environment. When adding Staging and Production to the Project Web Interface, the process includes all user accounts and settings. You can modify the settings and values for these environments after they are added.
+
+1. Log in to your Platform.sh account.
+2. From your project, click Master to view the environment information and settings.
+3. Click ![configure your project](/images/edit-project-gear.png) Configure environment.
```
Comment (chadwcarlson) [inline src/dedicated/overview/upgrading.md]: Broken image here as well

File: src/dedicated/overview/upgrading.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+4. Click the Users tab to review the user accounts and permission configurations.
+5. Add, delete, or update users, if needed.
+
+### Prepare variables
+
+When we convert your project to the new Project Web Interface, we add variables from Development environment to the Staging and Production environments. You can review, modify, and add variables through the current Project Web Interface prior to conversion.
+
+1. Log in to your Platform.sh account.
+2. From your project, click the master branch to view the environment information and settings.
+3. Click ![configure your project](/images/edit-project-gear.png) Configure environment.
```
Comment (chadwcarlson) [inline src/dedicated/overview/upgrading.md]: Here as well

File: src/dedicated/overview/upgrading.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+Once started the process usually takes less than an hour.  There should be no downtime on your production site, although you should not push any code to Git while the upgrade is in progress.
+
+When done, you can access review your project through the Project Web Interface.
+
+### (Optional) Migrate environment variables
+
+After conversion, you can manually migrate specific environment variables for Staging and Production.
+
+1. Open a terminal and checkout a branch in your local environment.
+2. List all environment variables:
```
Comment (chadwcarlson) [inline src/dedicated/overview/upgrading.md]: This spacing comes out a little funky. Maybe just place the command inline

> 2. List all environment variables: `platform variable:list`

File: src/dedicated/overview/upgrading.md (hunk 3 above)
Comment (Crell) [inline src/dedicated/overview/upgrading.md]:
```suggestion
3. Click ![configure your project](/images/dedicated/edit-project-gear.png) Configure environment.
```

File: src/dedicated/overview/upgrading.md (hunk 4 above)
Comment (Crell) [inline src/dedicated/overview/upgrading.md]:
```suggestion
3. Click ![configure your project](/images/dedicated/edit-project-gear.png) Configure environment.
```

**Pull request merged**